### ISSUE TITLE
Licenses for cljsjs/packages

### ISSUE THREAD

**Issue by shen-tian** [technomancy/lein-licenses#12]
I'm not seeing any packages for `cljsjs/packages` dependencies, e.g.

    cljsjs/react - 15.5.4-0 - unknown

Though (naively) looking at the [build scripts](https://github.com/cljsjs/packages/blob/master/react/build.boot) for those, they seem to include package information:

````
(task-options!
 pom  {:project     'cljsjs/react
       :version     +version+
       :description "A Javascript library for building user interfaces"
       :url         "http://facebook.github.io/react/"
       :scm         {:url "https://github.com/cljsjs/packages"}
       :license     {"BSD" "http://opensource.org/licenses/BSD-3-Clause"}})
````

What stops this from being picked up?

**Comment by technomancy** [comment]
Thanks for the bug report; there was a section in the code that made some faulty assumptions about the structure of the XML in the pom file. It's been fixed and released as 0.2.2.

**Comment by shen-tian** [comment]
Thanks!
